src/DotVVM.Framework/DependencyInjection/DotVVMServiceCollectionExtensions.cs

```diff
@@ -100,6 +101,8 @@ public static IServiceCollection RegisterDotVVMServices(IServiceCollection servi
                 o.TreeVisitors.Add(() => new LifecycleRequirementsAssigningVisitor());
             });
 
+            services.TryAddSingleton<IDotvvmCacheAdapter, DefaultDotvvmCacheAdapter>();
```

Comment:
I moved this line from the aspnetcore-specific and owin-specific ServiceCollection extensions, since both `IDotvvmCacheAdapter` and `DefaultDotvvmCacheAdapter` are in Framework. Not sure if this is an issue. @exyi, is this an issue?

Reply:
This should be ok. What's new for me is I didn't know that DefaultDotvvmCacheAdapter is used as the default for both platforms.